Add the Czech language

pchalupa opened the pull request:
We are all native Czech speakers and we would like to work on the Czech translation of the docs. We are co-workers from [STRV](https://github.com/strvcom) and We are using React on a daily basis.

Rados51 commented:
@pchalupa přidáš mě prosím také, rád vám pomůžu s překladem  🙌🏻

pchalupa commented:
> @pchalupa přidáš mě prosím také, rád vám pomůžu s překladem 🙌🏻

@Rados51 I would be happy to add you! 👏  Could you please briefly describe your experience with React and translation?

Rados51 commented:
@pchalupa

[Motion.page](https://motion.page) - created most of the code from the ground up alone  (custom components & hooks + Recoil).
As for translation, I was working as a translator for a Lego group in Kladno (Czech - Serbian translation).

pchalupa commented:
> @pchalupa
> 
> [Motion.page](https://motion.page) builder - created most of the code from the ground up alone (custom components & hooks + Recoil). As for translation, I was working as a translator for a Lego group in Kladno (Czech - Serbian translation).

@Rados51 welcome to the team! 🎉

gaearon commented:
As a part of fixing the permission sync process, we've changed how we track languages — please see https://github.com/reactjs/translations.react.dev/commit/63a10c4be3f7b10ffaf58fe144e2d5abca5dacc9. 
Could you please update this PR:

- Rebase on top of latest `main`, and add the language to `lang/langs.json` instead
- Include the list of maintainers into the PR comment body instead

Thank you, and sorry for the churn.

gaearon reviewed (changes requested):
As a part of fixing the permission sync process, we've changed how we track languages — please see https://github.com/reactjs/translations.react.dev/commit/63a10c4be3f7b10ffaf58fe144e2d5abca5dacc9. 
Could you please update this PR:

- Rebase on top of latest `main`, and add the language to `lang/langs.json` instead
- Include the list of maintainers into the PR comment body instead

Thank you, and sorry for the churn.

pchalupa commented:
> As a part of fixing the permission sync process, we've changed how we track languages — please see [63a10c4](https://github.com/reactjs/translations.react.dev/commit/63a10c4be3f7b10ffaf58fe144e2d5abca5dacc9). Could you please update this PR:
> 
> * Rebase on top of latest `main`, and add the language to `lang/langs.json` instead
> * Include the list of maintainers into the PR comment body instead
> 
> Thank you, and sorry for the churn.

@gaearon I have just updated this PR.
